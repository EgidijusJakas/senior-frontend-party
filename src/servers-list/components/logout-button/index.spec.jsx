import React from 'react';
import { shallow } from 'enzyme';
import { LogoutButton } from '.';

describe('Logout Button', () => {
    it('should render loader', () => {
        const renderedComponent = shallow(<LogoutButton onClick={jest.fn()} />);
        expect(renderedComponent).toMatchSnapshot();
    });
});
