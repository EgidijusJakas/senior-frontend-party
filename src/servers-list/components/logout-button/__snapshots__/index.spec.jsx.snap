// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logout Button should render loader 1`] = `
<styled.button
  onClick={[MockFunction]}
>
  Logout
</styled.button>
`;
